docs/DeveloperGuide.md

```diff
@@ -262,73 +262,227 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+- HR professional of a tech company with over 1000 employees with different skill sets and roles
```

Comment:
1000 might be a bit specific, we could consider just saying "many"?

Reply:
Thanks. Updated to use "many".

Reply:
Nice!